operator-common/src/main/java/io/strimzi/operator/common/operator/resource/AbstractNamespacedResourceOperator.java

```diff
@@ -248,6 +278,17 @@ protected Future<ReconcileResult<T>> internalUpdate(Reconciliation reconciliatio
         }
     }
 
+    protected Future<ReconcileResult<T>> internalPatch(Reconciliation reconciliation, String namespace, String name, T desired) {
```

Comment:
MAybe you can add some Javadocs to these methods even through they are not public?